<<<<<<<< HEAD:patch/kernel/archive/sunxi-6.18/patches.megous/private-6.18/0001-Make-microbuttons-on-Orange-Pi-PC-and-PC-2-work-as-p.patch
From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
========
From 80c12c413892885dce6c7a162c4f3fad729bf19e Mon Sep 17 00:00:00 2001
>>>>>>>> 09b789500459353b662b9482ae7912879c942f1a:patch/kernel/archive/sunxi-6.15/patches.megous/Make-microbuttons-on-Orange-Pi-PC-and-PC-2-work-as-power-off-bu.patch
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Fri, 18 Aug 2017 13:55:48 +0200
Subject: Make microbuttons on Orange Pi PC and PC 2 work as power off buttons

---
 arch/arm/boot/dts/allwinner/sun8i-h3-orangepi-one.dts    | 2 +-
 arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/allwinner/sun8i-h3-orangepi-one.dts b/arch/arm/boot/dts/allwinner/sun8i-h3-orangepi-one.dts
index 111111111111..222222222222 100644
--- a/arch/arm/boot/dts/allwinner/sun8i-h3-orangepi-one.dts
+++ b/arch/arm/boot/dts/allwinner/sun8i-h3-orangepi-one.dts
@@ -91,7 +91,7 @@ gpio-keys {
 
 		switch-4 {
 			label = "sw4";
-			linux,code = <BTN_0>;
+			linux,code = <KEY_POWER>;
 			gpios = <&r_pio 0 3 GPIO_ACTIVE_LOW>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
<<<<<<<< HEAD:patch/kernel/archive/sunxi-6.18/patches.megous/private-6.18/0001-Make-microbuttons-on-Orange-Pi-PC-and-PC-2-work-as-p.patch
index 111111111111..222222222222 100644
========
index f420acc6f49f..3d16a6efa2a7 100644
>>>>>>>> 09b789500459353b662b9482ae7912879c942f1a:patch/kernel/archive/sunxi-6.15/patches.megous/Make-microbuttons-on-Orange-Pi-PC-and-PC-2-work-as-power-off-bu.patch
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
@@ -60,7 +60,7 @@ gpio-keys {
 
 		key-sw4 {
 			label = "sw4";
-			linux,code = <BTN_0>;
+			linux,code = <KEY_POWER>;
 			gpios = <&r_pio 0 3 GPIO_ACTIVE_LOW>;
 			wakeup-source;
 		};
-- 
Armbian

